src/main/java/com/skynest/tests/files/Payload.java

```diff
@@ -0,0 +1,15 @@
+package com.skynest.tests.files;
+
+public class Payload {
```

Comment:
can you please try to make this class POJO? 

you can even leave this atm so we can compare the 2 approaches

you can even use Lombok to limit class size...

Reply:
Sure, will do a research and will try to make the class POJO.